content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/index.mdx

```diff
@@ -1,21 +1,14 @@
 ---
 title: 'Referential actions'
-metaTitle: 'Referential actions (Preview)'
+metaTitle: 'Referential actions'
 metaDescription: 'Referential actions let you define the update and delete behavior of related models on the database level'
 tocDepth: 2
-preview: true
 ---
 
 <TopBlock>
 
 In [2.26.0](https://github.com/prisma/prisma/releases/tag/2.26.0) and later, you can define **referential actions** on your relation fields. Referential actions determine what should happen to a record when a related record is deleted or updated.
```

Comment:
Should this maybe be rewritten to mention 3.0.x instead of when it was available as a preview feature flag?